Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Start with a base image containing Java runtime
+FROM openjdk:8-jdk-alpine
+
+# Add Maintainer Info
+LABEL maintainer="sladynnunes98@gmail.com"
+
+# Add a volume pointing to /tmp
+VOLUME /tmp
+
+# Make port 8080 available to the world outside this container
+EXPOSE 8080
+
+# The application's jar file
+ARG JAR_FILE=target/custom-distribution-service-0.0.1.jar
+
+# Add the application's jar to the container
+ADD ${JAR_FILE} custom-distribution-service.jar
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use COPY instead of ADD for files and folders](https://app.codacy.com/gh/jenkinsci/custom-distribution-service/pullRequest?prid=5632593)